lark/lark_validator.py

```diff
@@ -0,0 +1,265 @@
+from typing import Any, Dict, List
+
+from .exceptions import GrammarError
+from .grammar import TOKEN_DEFAULT_PRIORITY, RuleOptions
+from .lexer import Token
+from .load_grammar import eval_escaping
+from .tree import Tree
+
+class Definition:
+    def __init__(self, is_term, tree, params=(), options=None):
+        self.is_term = is_term
+        self.tree = tree
+        self.params = tuple(params)
+
+class LarkValidator:
+
+    @classmethod
+    def validate(cls, tree: Tree, options: Dict[str, Any] = {}):
```

Comment:
Why is there an `options` argument? I don't see it being used anywhere. Was the idea that it selects which checks to perform? I wouldn't be opposed to that being possible.

Reply:
That was left over from an earlier version of the validator.  I honestly don't remember what I did with it.  But you're right, it's dead code.  I think it ought to be removed, rather than leave it as a place to hang future functionality.  OK?

Reply:
Fixed by commit 20302ca65239bf952c3a10e4acd26b2f625b2a43.